src/game_battlealgorithm.cpp

```diff
@@ -918,103 +918,104 @@ bool Game_BattleAlgorithm::Normal::Execute() {
 
 	// Damage calculation
 	if (Rand::PercentChance(to_hit)) {
-		if (Rand::PercentChance(crit_chance)) {
-			critical_hit = true;
-		}
+		this->success = false;
+		return this->success;
+	}
```

Comment:
Because `this->success` is set to false here after a successful accuracy check, normal attacks almost always miss now (tested it myself). It looks like the check is supposed to be inverted, using `!Rand::PercentChance(to_hit)` fixes this.

Reply:
Fixed